.github/CODEOWNERS

```diff
@@ -942,6 +942,9 @@
 # ServiceLabel: %Spring App Discovery %Mgmt
 # ServiceOwners:                                                   @sunkun99
 
+# ServiceLabel: %EASM
```

Comment:
There's a bunch of things that need to be fixed for this:

- It is in the wrong place; please add this above in the SDK section, in alphabetical order.

- The form of this is incorrect.   There should be a seprarate PR and ServiceLabel section.  Please follow the existing entries as examples.

- Your path is too general.  You should have the specific library that you're responsible for in the path segment.  There will likely be other libraries under `/sdk/easm/` owned by different teams.

- You cannot just add a service label; these are centrally managed.  Please send email to "SDKGitHubNinjas" and request a new label.  Once approved, it will be added to the central list and pushed to all repositories.

Reply:
Do I use PRLabel instead of ServiceLabel here?

Reply:
This is the list of our approved names: https://github.com/Azure/azure-sdk/issues/5062. Maybe it can help us name the label?